pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -222,6 +224,7 @@ def get_strategic_model_unit_container():
 
 def create_model(df_sets, df_parameters, default={}):
     model = ConcreteModel()
+    salinity_dict={}
```

Comment:
salinity_dict={} as an argument in create_model caused the test to fail, so I moved it in the recent commit. I believe we dont need this in create model.